pytorch_lightning/loggers/base.py

```diff
@@ -21,6 +21,7 @@
 from functools import wraps
 from typing import Any, Callable, Dict, Iterable, List, Mapping, MutableMapping, Optional, Sequence, Tuple, Union
 
+import matplotlib.pyplot as plt
```

Comment:
I think you need to do a `_module_available("matplotlib")` check here.
matplotlib is a dependency that the user needs to install.